Base.rte/Scripts/AirBlast.lua

```diff
@@ -0,0 +1,29 @@
+function Create(self)
+	self.strength = self.Mass * self.Vel.Magnitude;
+	self.range = self.Lifetime * self.Vel.Magnitude;
+end
+function Update(self)
+	-- Run the effect on Update() to give other particles a chance to reach the target
+	for i = 1 , MovableMan:GetMOIDCount() - 1 do
+		local mo = MovableMan:GetMOFromID(i);
+		if mo and mo.PinStrength == 0 then
+			local dist = SceneMan:ShortestDistance(self.Pos, mo.Pos, SceneMan.SceneWrapsX);
+			if dist.Magnitude < self.range then
+				local strength = self.strength;
+				local strSumCheck = SceneMan:CastStrengthSumRay(self.Pos, self.Pos + dist, 3, 0);
+				if strSumCheck < strength then
+					strength = strength - strSumCheck;
```

Comment:
Given that strength is only used in one place (line 18, which has weirdass spacing btw) I see no reason to not just make line 18 be
`local forceVector = dist:SetMagnitude((strength - strengthSumCheck)/distFactor);`